### ISSUE TITLE
Issue with Instructions

### ISSUE THREAD

**Issue by themichaelbender** [pluralsight/PS-AutoLab-Env#85]
Currently the instructions on the main page have some gotchas.
First, .\Unattend-LabSetup (#8) does not exist.
Next, .\Unattend-Lab will not work with the leading .\ . It needs to run as just Unattend-Lab.

So I think it should read as follows:
Note: Unattend-Lab runs setup-lab, Run-Lab, and Validate-Lab all together for faster processing

If users don't look at the screenshot of commands and fully RTFM, they may try to run the command as written in 8 after changing to the proper config directory.

**Comment by theJasonHelmick** [comment]
Agreed.  Will update help files with new version currently under development

**Comment by Aamal** [comment]
Mine isn't running at all, help pls


PS C:\AutoLab\Configurations\Windows10> unattend-lab
unattend-lab : The 'unattend-lab' command was found in the module 'PSAutoLab', but the module could not be loaded. For more information, run 'Import-Module PSAutoLab'.
At line:1 char:1
+ unattend-lab
+ ~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (unattend-lab:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CouldNotAutoloadMatchingModule

**Comment by Aamal** [comment]
Where do I run that?

PS C:\AutoLab\Configurations> Import-module PSAutolab
Import-module : File C:\Program Files\WindowsPowerShell\Modules\PSAutolab\PSAutolab.psm1 cannot be loaded. The file C:\Program Files\WindowsPowerShell\Modules\PSAutolab\PSAutolab.psm1 is not digitally signed. You cannot run this script on
the current system. For more information about running scripts and setting execution policy, see about_Execution_Policies at http://go.microsoft.com/fwlink/?LinkID=135170.
At line:1 char:1
+ Import-module PSAutolab
+ ~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : SecurityError: (:) [Import-Module], PSSecurityException
    + FullyQualifiedErrorId : UnauthorizedAccess,Microsoft.PowerShell.Commands.ImportModuleCommand


Aamal Abed
Technical Analyst, Information Technology
T:  403.736.4351     F: 403.250.7492
500, 1212 31 Ave. NE, Calgary AB T2E 7S8
Find us on Facebook<http://www.facebook.com/> @ChinookFinancial & @FirstCalgary
Connectfirstcu.com<http://www.connectfirstcu.com/>

[Connect-First-colour]<http://www.connectfirstcu.com/>

**Comment by jdhitsolutions** [comment]
You can run import-module at any prompt. But the error message helps explain. If you run:

`Get-ExecutionPolicy`

You will get a policy called AllSigned.  Assuming you are in an elevated session run this command:

`Set-ExecutionPolicy RemoteSigned`

If prompted, go ahead and say ok. Then try importing the module. If you don't get any errors, change to the configuration directory and run the unattend-lab command.

**Comment by Aamal** [comment]
My policy is set to RemoteSigned
Still got an error. Same Not digitally signed.





PS C:\> Get-ExecutionPolicy
RemoteSigned

PS C:\> Set-ExecutionPolicy RemoteSigned

PS C:\> Import-module PSAutolab
Import-module : File C:\Program Files\WindowsPowerShell\Modules\PSAutolab\PSAutolab.psm1 cannot be loaded. The file C:\Program Files\WindowsPowerShell\Modules\PSAutolab\PSAutolab.psm1 is not digitally signed.
You cannot run this script on the current system. For more information about running scripts and setting execution policy, see about_Execution_Policies at http://go.microsoft.com/fwlink/?LinkID=135170.
At line:1 char:1
+ Import-module PSAutolab
+ ~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : SecurityError: (:) [Import-Module], PSSecurityException
    + FullyQualifiedErrorId : UnauthorizedAccess,Microsoft.PowerShell.Commands.ImportModuleCommand

PS C:\>



Aamal Abed
Technical Analyst, Information Technology
T:  403.736.4351     F: 403.250.7492
500, 1212 31 Ave. NE, Calgary AB T2E 7S8
Find us on Facebook<http://www.facebook.com/> @ChinookFinancial & @FirstCalgary
Connectfirstcu.com<http://www.connectfirstcu.com/>

[Connect-First-colour]<http://www.connectfirstcu.com/>

**Comment by jdhitsolutions** [comment]
Then re-run Set-ExecutionPolicy and use a value of ByPass per the Readme instructions.

**Comment by Aamal** [comment]
Cut the folder PSAutoLab

Pasted it on my desktop

Unblocked the 2 txt files
Moved it to C:\Windows\System32\WindowsPowerShell\v1.0\Modules

Ran the import module

Success

Run the unattend-lab

WORKING ☺

Aamal Abed

**Comment by Aamal** [comment]
Is it supposed to show up in the HyperV mgr?  it isn’t and I am following as we speak the vid and it doesn’t say the name of the VM?

So sorry

Aamal Abed

**Comment by jdhitsolutions** [comment]
You should not have had to do any of that. What txt files did you have to unblock?

**Comment by jdhitsolutions** [comment]
If you moved the Autolab folder after the initial setup, building VMs might not work because the path has changed.  We need to get this to work with your original folder, presumably C:\Autolab.

**Comment by Aamal** [comment]
So I need to uninstall and undo what I did than move the folder back to the original location and redo the work.
How do I undo? So new to this

Aamal Abed

**Comment by jdhitsolutions** [comment]
Just move the folder back from where you moved it. But if the setup process works, you might just live with it. I assumed there would be a problem but I could be mistaken. And for the sake of reading issues, there's no need to reply with the previous message.

**Comment by jeffhongxiao** [comment]
`Set-ExecutionPolicy Bypass -Scope CurrentUser` solves the @Aamal 's issue:

My policy is set to RemoteSigned Still got an error. Same Not digitally signed.

**Comment by jdhitsolutions** [comment]
This project is now under new ownership. The 3.x version line is obsolete. Version 4.0.0 has been published to the PowerShell Gallery with new updates, configuration, and documentation. It is expected that this version will resolve most if not all of the problems you were experiencing. Read [Updating](https://github.com/pluralsight/PS-AutoLab-Env/blob/master/Updating.md) for more information.

No further action will be taken on this issue and it will most likely be closed at some point. If you still have the same issue with the new version, then go ahead and open a new issue.
